e2e/contracts/testprototype/TestPrototype.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I'm thinking about centralizing all contracts used in protocol in `pkg/contracts`, like TestDappV2 for example.

Maybe we could move this one there unless we deem this is too scoped to e2e